variation/classifiers/classifier.py

```diff
@@ -2,20 +2,44 @@
 from abc import ABC, abstractmethod
 from typing import List, Optional
 
-from variation.schemas.classification_response_schema import Classification,\
-    ClassificationType
-from variation.schemas.token_response_schema import Token
+from variation.schemas.classification_response_schema import Classification
+from variation.schemas.token_response_schema import Token, TokenType
 
 
 class Classifier(ABC):
     """The Classifier class."""
 
     @abstractmethod
     def match(self, tokens: List[Token]) -> Optional[Classification]:
-        """Return the classification from a list of token matches."""
-        raise NotImplementedError
+        """Return the classification from a list of token matches.
+
+        :param tokens: List of ordered tokens that are exact match candidates for a
+            given classification
+        :return: A classification for the list of matched tokens
+        """
 
     @abstractmethod
-    def classification_type(self) -> ClassificationType:
-        """Return the classification type."""
-        raise NotImplementedError
```

Comment:
some of the Classifier child classes still implement this method, but it looks like it isn't used anywhere anymore?

Reply:
Ope forgot to remove